### PR TITLE
Test runs now generate a separate directory

### PR DESCRIPTION
**Pull request by andxu282**
### Summary

This PR addresses Jira ticket [CISLUNAR-XX](https://cislunarsoftware.atlassian.net/browse/CISLUNAR-140)     
- center find feature test now generates a separate directory based on the current unix time
- all results file are placed in this directory

### Comments
- [x] Add an x between the brackets if you commented your code well!

### CODE REVIEW

File: OpticalNavigation/tests/test_center_finding.py [hunk 1]
```diff
@@ -95,7 +101,18 @@ def get_results(self, dir, results_file, st_gn, pixel=True):
                 results.append(perf_values)
 
         # Writing performance values to csv file
-        with open(results_file, "w", newline="") as csvfile:
+        os.makedirs(
```
Comment (tmf97) [inline OpticalNavigation/tests/test_center_finding.py:104]: Nit: when I run `pytest` from the root directory, the results end up in `FlightSoftware/tests/`, is this the intended behavior?

File: OpticalNavigation/tests/test_center_finding.py [hunk 2]
```diff
@@ -95,7 +101,18 @@ def get_results(self, dir, results_file, st_gn, pixel=True):
...
+            self.cwd + "/tests/center_find_test_run_" + str(self.now), exist_ok=True
+        )
+        with open(
+            self.cwd
+            + "/tests/center_find_test_run_"
+            + str(self.now)
+            + "/"
+            + results_file,
+            "w",
+            newline="",
```
Comment (tmf97) [inline OpticalNavigation/tests/test_center_finding.py:114]: I'd recommend using f-strings for stuff like this: https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python

**Review by tmf97 — COMMENTED** [review]
Looks solid to me and runs on my machine. Gotta love being organized

File: OpticalNavigation/tests/test_center_finding.py (hunk 1 above)
Comment (sjz38) [reply OpticalNavigation/tests/test_center_finding.py:104]: I would recommend putting it in `OpticalNavigation/tests`

**Review by sjz38 — CHANGES_REQUESTED** [review]
Once you change the small things we mentioned, I'll be good to approve!

**Review by tmf97 — APPROVED** [review]

**Review by sjz38 — APPROVED** [review]
👍I'll merge since Andrew left

**Pull request merged**